docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -16,6 +16,7 @@ internal static void RunAll()
 
                 if (sample != null)
                 {
+                    Console.WriteLine(type.Name);
```

Comment:
This will print the name of the class before the example is run in the console. That way we know which example gave which results when we get all the outputs.